app/styles/core-v2/_shims.less

```diff
@@ -0,0 +1,63 @@
+/* cyrillic-ext */
```

Comment:
What is the goal of this file?

Reply:
It forces Bold/700 font-weights to load the SemiBold font. This way we don't have to chase all Bold usages right now.

Reply:
So it will be impossible to set bold in the soft after that

Reply:
Yep !